src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -172,7 +177,7 @@ public void setPythonStartupArgs(String pythonStartupArgs) {
   }
 
   public void setJvmFlags(List<String> jvmFlags) {
-    this.jvmFlags = jvmFlags;
+    this.jvmFlags = jvmFlags != null ? ImmutableList.copyOf(jvmFlags) : null;
```

Comment:
I wonder if we should do safer copying of lists everywhere?

It's a good idea, but is the expectation that a user would change their environment after setting this?